docs/markdown/Using Pants/using-pants-in-ci.md

```diff
@@ -15,9 +15,17 @@ Directories to cache
 
 In your CI's config file, we recommend caching these directories:
 
-- `$HOME/.cache/pants/setup`: the initial bootstrapping of Pants.
-- `$HOME/.cache/pants/named_caches`: caches of tools like pip and PEX.
-- `$HOME/.cache/pants/lmdb_store`: cached content for prior Pants runs, e.g. prior test results.
+- `$HOME/.cache/pants/setup`: The initial bootstrapping of Pants.
+- `$HOME/.cache/pants/named_caches`: Caches used by underlying tools like Pip and PEX.
+
+If you're not using a fine-grained [remote caching](doc:remote-caching-execution) service, 
+such as the one provided by [Toolchain](https://toolchain.com/), the lead sponsor of Pants, 
+then you may also want to preserve the local Pants cache:
```

Comment:
Same comment as https://github.com/pantsbuild/example-python/pull/104#discussion_r944730976

Reply:
Removed this mention in favor of a link back to the other page.